src/bindings/js/node/include/helper.hpp

```diff
@@ -99,12 +109,39 @@ template <>
 Napi::String cpp_to_js<ov::element::Type_t, Napi::String>(const Napi::CallbackInfo& info,
                                                           const ov::element::Type_t type);
 
-const std::map<std::string, ov::element::Type_t> element_type_map = {{"i8", ov::element::Type_t::i8},
-                                                                     {"u8", ov::element::Type_t::u8},
-                                                                     {"i16", ov::element::Type_t::i16},
-                                                                     {"u16", ov::element::Type_t::u16},
-                                                                     {"i32", ov::element::Type_t::i32},
-                                                                     {"u32", ov::element::Type_t::u32},
-                                                                     {"f32", ov::element::Type_t::f32},
-                                                                     {"f64", ov::element::Type_t::f64},
-                                                                     {"i64", ov::element::Type_t::i64}};
+template <typename KeyType>
+ov::Tensor get_request_tensor(ov::InferRequest infer_request, KeyType key);
```

Comment:
Is template required? Maybe just make set of overloaded functions and not use template.